### PR TITLE
Fixing some light/dark mode issues lying around the site.

### PR DESCRIPTION
**Pull request by eti7075**
- Added a variable background color for the theme selector icon to match the navbar color scheme in dark mode.
- Changed the gutter color and scroller color for the scroll bar in the inspector panel table during dark mode usage.
- Updated the :root background color scheme to be more consistent. Now, regardless of the user's _browser_ dark/light preference, the site root will have the same background, resulting in a consistent map shade.

### CODE REVIEW

File: site/src/InspectorPanel.css [hunk 1]
```diff
@@ -61,4 +61,7 @@
 
   box-shadow: 0 2px 4px black;
   color: white;
+
+  scrollbar-color: var(--ifm-color-secondary-darkest)
```
Comment (eti7075) [inline site/src/InspectorPanel.css:65]: Only substantive change. This lies within the .theme-dark .inspector-panel property definition. Other changes are formatting

File: site/src/InspectorPanel.jsx [hunk 2]
```diff
@@ -23,7 +23,11 @@ function InspectorPanel({ entity }) {
         </tbody>
       </table>
       <p>
-        <a href="https://docs.overturemaps.org/schema/" target="_blank" rel="noreferrer noopener">
+        <a
+          href="https://docs.overturemaps.org/schema/"
```
Comment (eti7075) [inline site/src/InspectorPanel.jsx:27]: Formatting

File: site/src/Map.jsx [hunk 3]
```diff
@@ -327,7 +325,10 @@ export default function Map({ mode, mapEntity, setMapEntity, setZoom }) {
             <InspectorPanel entity={mapEntity} />
           )}
 
-          <ThemeSelector visibleThemes={setVisibleThemes}></ThemeSelector>
+          <ThemeSelector
+            mode={mode}
```
Comment (eti7075) [inline site/src/Map.jsx:329]: Adds mode state to theme selector, also includes formatting changes

File: site/src/ThemeSelector.css [hunk 4]
```diff
@@ -33,4 +33,9 @@ svg.icon-layers > path.primary {
 
 svg.icon-layers > path.secondary {
   fill: cornflowerblue;
-}
\ No newline at end of file
+}
+
+svg.icon-layers-dark {
```
Comment (eti7075) [inline site/src/ThemeSelector.css:38]: only substantive change, adding a adaptive background coloring and border radius to match the light mode shape. All other changes in file are formatting.

File: site/src/ThemeSelector.jsx [hunk 5]
```diff
@@ -24,7 +24,11 @@ function ThemeSelector({ visibleThemes }) {
   return (
     <div className="dropdown dropdown--hoverable theme-selector tour-layers">
       <div className="layer-control">
-        <LayerIcon />
+        <LayerIcon
+          className={`icon-layers ${
```
Comment (eti7075) [inline site/src/ThemeSelector.jsx:28]: adding classname to icon for css

File: site/src/index.css [hunk 6]
```diff
@@ -92,3 +92,9 @@ div.theme-light {
   color: #213547;
   background-color: #ffffff;
 }
+
+@media (prefers-color-scheme: dark) {
```
Comment (eti7075) [inline site/src/index.css:96]: updating the background during dark mode browser usage

**Review by Bonkles — APPROVED** [review]

**Pull request merged**